### ISSUE TITLE
没法动态控制选取框高度和长度，只能控制形状

### ISSUE THREAD

**Issue by iron-box-of-an-island** [aesoper101/vue3-colorpicker#51]
而且chrome模式的回显有问题，无法回显该有的颜色，全是白色
![image](https://github.com/aesoper101/vue3-colorpicker/assets/131111180/fa39bc80-ab44-4a02-af02-340ba4eb4698)

**Comment by alonehover** [comment]
Thank you for your feedback. The echo issue will be fixed in version 2.1.4. 
If you need to customize the control height and width, you can use the style override method.

For example:
````
<style scoped>
:deep(.vc-color-wrap) {
  height: 20px;
  width: 100px;
}
</style>
````

**Comment by iron-box-of-an-island** [comment]
well，thanks
